subhub/customer.py

```diff
@@ -113,7 +124,11 @@ def has_existing_plan(user: stripe.Customer, plan_id: str) -> bool:
     :return: True if user has existing plan, otherwise False
     """
     customer = stripe.Customer.retrieve(user.id)
-    for item in customer["subscriptions"]["data"]:
-        if item["plan"]["id"] == plan_id and item["status"] in ["active", "trialing"]:
-            return True
+    if customer.get("subscriptions"):
+        for item in customer["subscriptions"]["data"]:
+            if item["plan"]["id"] == plan_id and item["status"] in [
```

Comment:
This is, I believe a terse way to do it.
`return [x for x in input_dict if customer["subscriptions"]["data"] in ["active", "trialing"]] is not None`

Reply:
only thing I would add is that I prefer tuples when comparing for the existence of an item in a series ie `in ("active", "trailing")`.  But that's personal choice.  The reason I do that is because tuples are immutable and lists aren't; so it conveys intent a little more clearly.

Reply:
Good point, immutability ftw.

Reply:
The example you wrote out does not retrieve the data correctly, I understand using list comprehension may seem like a better want to do this but for now I am going to leave the existing structure in place.

Reply:
cool!

Reply:
I wasn't talking about the list comprehension; but rather the list of items you were comparing to